packages/pn-personagiuridica-webapp/src/components/IntegrazioneApi/NewPublicKey/ShowPublicKeyParams.tsx

```diff
@@ -0,0 +1,93 @@
+import { useTranslation } from "react-i18next";
+import { useNavigate } from "react-router-dom";
+
+import { CopyToClipboardButton } from '@pagopa/mui-italia';
+import VpnKeyIcon from '@mui/icons-material/VpnKey';
+import AssignmentIndIcon from '@mui/icons-material/AssignmentInd';
+import { InputAdornment, InputLabel, TextField, Typography } from '@mui/material';
+import { Box, Stack } from "@mui/system";
+
+import { BffPublicKeyResponse } from "../../../generated-client/pg-apikeys";
+import * as routes from '../../../navigation/routes.const';
+import NewPublicKeyCard from './NewPublicKeyCard';
+
+type Props = {
+    params: BffPublicKeyResponse | undefined;
+};
+
+const ShowPublicKeyParams: React.FC<Props> = ({params}) => {
+    const { t } = useTranslation(['integrazioneApi', 'common']);
+    const navigate = useNavigate();
+
+    const handleSubmit = () => navigate(routes.INTEGRAZIONE_API);
+
+    return (
+        <form onSubmit={handleSubmit} data-testid="publicKeyDataInsertForm">
```

Comment:
Move the form under the card tag

Reply:
Moving the form inside the card makes the form submission not working anymore!

Reply:
Already discussed above.